docs/shadow_config_spec.md

```diff
@@ -125,7 +122,7 @@ General experiment settings.
 
 #### `general.bootstrap_end_time`
 
-Default: "0 sec"  
+Default: "0 sec"
 Type: String OR Integer
```

Comment:
I don't think we want to remove the spaces here, otherwise the rendered html will show as

```text
Default: "0 sec" Type: String OR Integer
```

instead of

```text
Default: "0 sec"
Type: String OR Integer
```

Reply:
Ahh yes, thanks! I'll undo those, and make sure to curse my VS Code setting while doing it :)